src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -1257,7 +1257,7 @@ void AlgorithmPaintToolBox::updateMouseInteraction() //Apply the current interac
 dtkPlugin* AlgorithmPaintToolBox::plugin()
 {
     medPluginManager* pm = medPluginManager::instance();
-    dtkPlugin* plugin = pm->plugin ( "mscAlgorithmPaint" );
+    dtkPlugin* plugin = pm->plugin ( "Algorithm Paint" );
```

Comment:
It was the plugin "name()" in Paint plugin. I don't mind changing it, but it was just to debug it.